.gitmodules

```diff
@@ -0,0 +1,22 @@
+[submodule "lib/forge-std"]
+	branch = "v1"
+	path = "lib/forge-std"
+	url = "https://github.com/foundry-rs/forge-std"
+[submodule "lib/prb-test"]
+	branch = "release-v0"
+	path = lib/prb-test
+	url = https://github.com/PaulRBerg/prb-test
+[submodule "lib/openzeppelin-contracts"]
+	path = lib/openzeppelin-contracts
+	url = https://github.com/OpenZeppelin/openzeppelin-contracts
+[submodule "lib/osx"]
+	path = lib/osx
+	url = https://github.com/aragon/osx
+[submodule "lib/openzeppelin-contracts-upgradeable"]
+	branch = "release-v4.9"
+	path = lib/openzeppelin-contracts-upgradeable
+	url = https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable
+[submodule "lib/ens-contracts"]
+	path = lib/ens-contracts
+	url = https://github.com/ensdomains/ens-contracts
```

Comment:
Probably not needed (we should remove them too on the hardhat template). Feel free to do this in another task.
Generally, let's only include what is strictly needed.

Reply:
these are actually needed, when you import OSx as a gitsubmodule the ens contracts are not bundled with it